implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/types/DynatraceDistributionSummary.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2022 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.dynatrace.types;
+
+import io.micrometer.core.instrument.AbstractMeter;
+import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.distribution.HistogramSnapshot;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.TimeUnit;
+
+public class DynatraceDistributionSummary extends AbstractMeter implements DistributionSummary, DynatraceSummarySnapshotSupport {
+    private final DynatraceSummary summary = new DynatraceSummary();
+    private static final Logger LOGGER = LoggerFactory.getLogger(DynatraceDistributionSummary.class.getName());
+
+    public DynatraceDistributionSummary(Id id) {
+        super(id);
+    }
+
+    @Override
+    public void record(double amount) {
+        summary.recordNonNegative(amount);
+    }
+
+    @Override
+    public long count() {
+        return summary.getCount();
+    }
+
+    @Override
+    public double totalAmount() {
+        return summary.getTotal();
+    }
+
+    @Override
+    public double max() {
+        return summary.getMax();
+    }
+
+    public double min() {
```

Comment:
There is no usage of it currently, but its probably because the other DistributionSummaries and Timers don't even offer min. I added it because since we offer min, we also might want to offer it in this interface.